app/queries/fulltext/posts/search_query.rb

```diff
@@ -0,0 +1,21 @@
+module Fulltext
+  module Posts
+    class SearchQuery
+      DEFAULT_RECORDS_COUNT = 100
+
+      def initialize(search_criteria)
+        @search_criteria = search_criteria
+      end
+
+      def call
+        return Post.first(DEFAULT_RECORDS_COUNT) unless search_criteria.present?
+
+        Post.search_full_text(search_criteria)
```

Comment:
we don't limit here? :P

Reply:
I wanted to avoid limiting the query so we can get better benchmarks of the performance. Adding limit at SQL level may affect the results. What do you think?

Reply:
makes sense